docs/docs/installation/configuration/namespace-keptn.md

```diff
@@ -0,0 +1,78 @@
+---
+comments: true
+---
+
+# Namespaces and Keptn
+
+This page discusses how to restrict the namespaces
+where Keptn and the
+[lifecycle-operator](../../../components/lifecycle-operator)
+run.
+
+For information about how to allocate Keptn resources
+across namespaces, see
+[Namespaces and resources](namespace-resources.md).
+
+## Default behavior
+
+Keptn must be installed on its own namespace
+that does not run any other components,
+especially any application deployment.
+
+By default, Keptn lifecycle orchestration is enabled
+for all namespaces except the followings:
+
+- `kube-system`
+- `kube-public`
+- `kube-node-lease`
+- `cert-manager`
+- `keptn-system`
```

Comment:
I guess we can leave this out if we have the <Keptn installation namespace> in the list